programs/summerofcode/2024.md

```diff
@@ -31,6 +31,24 @@ You can find the project ideas from previous year [here](./2023.md).
 
 ### Proposals
 
+#### CNCF GOSST
+
+##### CNCF and Google Open Source Security Team GSoC Collaboration - Enhancing Security Across CNCF Ecosystem
+
+- Description: This project is a collaborative effort between the CNCF and Google's Open Source Security Team to improve security practices across various CNCF projects. The focus is identifying and addressing security vulnerabilities, integrating security tools like OSS-Fuzz, and enhancing build and release security processes. The goal is to get all CNCF projects to use scorecards (focusing on graduated/incubating projects first) and to remediate some of the findings.
+- Expected Outcome:
+  * All graduated and incubating CNCF projects using OpenSSF Scorecards to assess and enhance their security postures. Stretch goal: all (including sandbox) CNCF projects using OpenSFF Scorecards.
+    * Remediation of identified vulnerabilities based on scorecard findings
+    * Where CNCF projects are already using [OpenSSF Scorecard](https://securityscorecards.dev/), improved scores (remediating [various risk assessments](https://securityscorecards.dev/#the-checks)
+  * Integration or enhancement of fuzzing with [OSS-Fuzz](https://google.github.io/oss-fuzz/) for CNCF projects
+  * Improved build/release security by automating builds and releases, added build provenance, signing, and improved reproducibility
+- Recommended Skills: Security analysis, CI/CD practices, programming (preferably Go), knowledge of CNCF projects.
```

Comment:
Thanks for tagging me. 

I like the idea, but the details of implementation, execution, and resulting impact must be dutifully weighed for impact versus toil on cloud native projects.

I am happy to discuss this more in depth

Reply:
Thank you so much for your feedback!

> I like the idea, but the details of implementation, execution, and resulting impact must be dutifully weighed for impact versus toil on cloud native projects.

As i mentioned above, I'd like to merge this in as is for timing, but am happy to continue to massage the proposal.

> I am happy to discuss this more in depth

Thanks, I'll reach out offline.